web/util.py

```diff
@@ -0,0 +1,10 @@
+from flask import flash, redirect
+
+def allowed_file(filename: str, allowed_ext: object):
```

Comment:
Are we using `allowed_file` anywhere?

Let's get rid of it.

Reply:
My thought was it could be used for future upgrades (allowing uploading gcode) hence keeping it, but can get rid of it and restore it if we need it later :)